adapters/handlers/graphql/local/get/class_builder_fields.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/weaviate/weaviate/entities/schema"
 	"github.com/weaviate/weaviate/entities/search"
 	"github.com/weaviate/weaviate/entities/searchparams"
-	"github.com/weaviate/weaviate/usecases/traverser"
+	"github.com/weaviate/weaviate/types/dto"
```

Comment:
As @aliszka mentioned on Slack, it seems the new root level package `types` serves the same purpose as the existing `entities` pkg, so I'd move `types/dto` to `entities/dto`.